src/QueueReceiver.Infrastructure/Data/ApplicationDbContext.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.EntityFrameworkCore;
+using QueueReceiver.Core.Models;
+
+namespace QueueReceiver.Infrastructure.Data
+{
+    public class ApplicationDbContext : DbContext
+    {
+        public ApplicationDbContext(
```

Comment:
Weird formatting, should be:

        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options)
        {
        }